app/Jobs/RestartProxyJob.php

```
@@ -19,27 +22,139 @@ class RestartProxyJob implements ShouldBeEncrypted, ShouldQueue
         }
     }
+
+    /**
+     * Build combined stop + start commands for proxy restart.
+     * This creates a single command sequence that shows all logs in one activity.
+     */
+    private function buildRestartCommands(): array
+    {
+        $proxyType = $this->server->proxyType();
+        $containerName = $this->server->isSwarm() ? 'coolify-proxy_traefik' : 'coolify-proxy';
+        $proxy_path = $this->server->proxyPath();
```

PHPMD (2.15.0)

The variable $proxy_path is not named in camelCase. (undefined)

(CamelCaseVariableName)